Classes/View/PreviewView_7_2.php

```diff
@@ -0,0 +1,87 @@
+<?php
+namespace FluidTYPO3\Flux\View;
+
+/*
+ * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+use FluidTYPO3\Flux\Form\Container\Column;
+use FluidTYPO3\Flux\Service\ContentService;
+use TYPO3\CMS\Backend\View\PageLayoutView;
+
+/**
+ * @package Flux
+ */
+class PreviewView_7_2 extends PreviewView {
```

Comment:
Correct me if I'm wrong, but: is it not possible to combine this markup (the DOM structure as well as class names) generated by both these functions, in the default PreviewView? As far as I can tell from reading, those added containers should not cause problems. That way you wouldn't need this class, the Preview_7_2 class or the switching that goes on in `ext_localconf.php`.